pkg/deploy/predeploy.go

```diff
@@ -490,14 +486,14 @@ func (d *deployer) ensureSecretKey(ctx context.Context, kv keyvault.Manager, sec
 	})
 }
 
-func (d *deployer) restartOldScalesets(ctx context.Context, resourceGroupName string) error {
-	scalesets, err := d.vmss.List(ctx, resourceGroupName)
+func (d *deployer) restartOldRPScalesets(ctx context.Context) error {
```

Comment:
I'm not sure you need to rename the functions to include `RP`, to it makes sense to keep them the same as they were in case we need to reuse them in the future.

Reply:
Reverted the function rename.